services/.env.example

```
@@ -132,8 +132,11 @@ USERCONFIG_121_SERVICE_PASSWORD_PROGRAM_ADMIN=password
 USERCONFIG_121_SERVICE_EMAIL_USER_VIEW=view-user@example.org
 USERCONFIG_121_SERVICE_PASSWORD_USER_VIEW=password
 
-USERCONFIG_121_SERVICE_EMAIL_USER_KOBO=kobo-user@example.org
-USERCONFIG_121_SERVICE_PASSWORD_USER_KOBO=password
+USERCONFIG_121_SERVICE_EMAIL_USER_KOBO_REGISTRATION=kobo+registration_country@121.global
+USERCONFIG_121_SERVICE_PASSWORD_USER_KOBO_REGISTRATION=password
+
+USERCONFIG_121_SERVICE_EMAIL_USER_KOBO_VALIDATION=kobo+validation_country@121.global
+USERCONFIG_121_SERVICE_PASSWORD_USER_KOBO_VALIDATION=password
```

```suggestion
USERCONFIG_121_SERVICE_EMAIL_USER_KOBO_REGISTRATION=kobo+registration_country@example.org
USERCONFIG_121_SERVICE_PASSWORD_USER_KOBO_REGISTRATION=password

USERCONFIG_121_SERVICE_EMAIL_USER_KOBO_VALIDATION=kobo+validation_country@example.org
USERCONFIG_121_SERVICE_PASSWORD_USER_KOBO_VALIDATION=password
```

This file is for 'realistic' example-values; Not for actual existing email-addresses. 

In the actual running instance, these ENV-variables can be set appropriately.

---

@elwinschmitz Ok thanks! This is the convention we set for the kobo registration and validation emailaddress, where "country" needs to be replaced by the "hostname", eg, ethiopia, kenya, etc. Still, you think we should change this?

---

Yes. My main point is about the "real world values" v.s. "example values". (So the `@121.global`-part only!)

In our (open source) codebase we should have 'generic' code, no "implementation specific"-values. (Or at least try to have as little as possible)